emote/callbacks.py

```diff
@@ -359,7 +359,7 @@ class Checkpointer(Callback):
     callbacks supplied in the constructor.
 
     :param callbacks (List[Callback]): A list of callbacks the should be saved.
-    :param path (str): The path to where the checkpoint should be stored.
+    :param run_root (str): The root path to where the checkpoint should be stored.
```

Comment:
You need to add the new param too! And this needs to say where "run artifacts should be stored" or similar.